docs/setup/settings.asciidoc

```diff
@@ -513,7 +513,7 @@ In addition to this setting, trusted certificates may be specified via <<server-
 | `server.ssl.cipherSuites:`
  | Details on the format, and the valid options, are available via the
 https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT[OpenSSL cipher list format documentation].
```

Comment:
I believe that OpenSSL is bumped to `1.1.1` as part of this release as well:

```suggestion
https://www.openssl.org/docs/man1.1.1/apps/ciphers.html#CIPHER-LIST-FORMAT[OpenSSL cipher list format documentation].
```

Reply:
Fixed. The path was a bit different for the 1.1.1 docs, so I pushed a new commit